### PR TITLE
Drive Code Update

### PR DESCRIPTION
**Pull request by blazeboy75**

### CODE REVIEW

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/StrafeDrive.java [hunk 1]
```diff
@@ -23,6 +27,10 @@ public void init() {
 
         // Do all init stuff
         // TODO: ADD INITS THAT YOU NEED
+
+        //Speed Variables
```
Comment (teowelton) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/StrafeDrive.java]: remove comment

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/StrafeDrive.java [hunk 2]
```diff
@@ -32,6 +40,27 @@ public void init() {
     // init_loop() - Runs continuously until the driver hits play
     @Override
     public void init_loop() {
```
Comment (teowelton) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/StrafeDrive.java:38]: add comments (sorry qcr2 requires it)

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/StrafeDrive.java [hunk 3]
```diff
@@ -32,6 +40,27 @@ public void init() {
...
+        if (gamepad1.a == true) {
+
+            if (gamepad1.dpad_down == true) {
+                tSpeed -= 1;
+            } else if (gamepad1.dpad_up == true) {
+                tSpeed += 1;
+            } else if (gamepad1.b == true) {
```
Comment (teowelton) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/StrafeDrive.java]: dear lord fix your indentation. Dont you fkn say I'm acting like Milo, This is just horrendous lmao

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/StrafeDrive.java [hunk 4]
```diff
@@ -32,6 +40,27 @@ public void init() {
...
+
+                if (gamepad1.dpad_down == true) {
+                    dSpeed -= 1;
+                } else if (gamepad1.dpad_up) {
+                    dSpeed += 1;
+                } else if (gamepad1.x == true) {
```
Comment (teowelton) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/StrafeDrive.java]: Same here please fix your goddamn indentation

**Review by teowelton — CHANGES_REQUESTED** [review]

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/StrafeDrive.java (hunk 4 above)
Comment (olivia-banks) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/StrafeDrive.java]: Gracious professionalism.

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/StrafeDrive.java (hunk 3 above)
Comment (olivia-banks) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/StrafeDrive.java]: Gracious professionalism. /s

I agree though.

**Review by teowelton — APPROVED** [review]
This is Milo, I agree to a degree (hey, that rhymes)!

**Pull request merged**